performance/benchmarks/__init__.py

```diff
@@ -227,6 +227,18 @@ def BM_Telco(python, options):
 
 
 @VersionRange()
+def BM_Telco_Fractions(python, options):
+    bm_path = Relative("bm_telco_fractions.py")
+    return run_perf_script(python, options, bm_path, extra_args=['fractions'])
+
+
+@VersionRange()
+def BM_Telco_Decimal(python, options):
+    bm_path = Relative("bm_telco_fractions.py")
+    return run_perf_script(python, options, bm_path, extra_args=['decimal'])
```

Comment:
Agreed, unless you want to point out the speed difference.  But that was my question on bugs.python.org: What is the specific goal of this benchmark? :)